src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Internal/Controllers/PartyController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Altinn.AccessManagement.Api.Internal.Extensions;
+using Altinn.AccessManagement.Core.Models.Party;
+using Altinn.AccessMgmt.Persistence.Core.Models;
+using Altinn.AccessMgmt.Persistence.Data;
+using Altinn.AccessMgmt.Persistence.Services.Contracts;
+using Altinn.Authorization.Api.Contracts.Party;
+using Altinn.Authorization.ProblemDetails;
+using Azure.Core;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.AccessManagement.Controllers
+{
+    // TODO: Add proper authorization and authentication attributes to the controller
```

Comment:
ToDo her må absolutt løses.
Må kreve AccessToken med platform som issuer og helst også app claim sjekk for Authentication.

Reply:
fixed